src/diffusion_for_multi_scale_molecular_dynamics/models/score_networks/diffusion_mace_score_network.py

```diff
@@ -88,12 +99,12 @@ def __init__(self, hyper_params: DiffusionMACEScoreNetworkParameters):
                 hyper_params.interaction_cls_first
             ],
             num_interactions=hyper_params.num_interactions,
-            num_elements=hyper_params.number_of_elements,
+            num_elements=hyper_params.num_atom_types
```

Comment:
I think `num_elements` will be confusing. I suggest renaming this parameter `num_classes` to align with what is done elsewhere.